elephant/signal_processing.py

```diff
@@ -136,6 +137,129 @@ def zscore(signal, inplace=True):
         return result
 
 
+def cross_correlation_function(signal, ch_pairs, env=False, nlags=None):
+
+    """
+    Computes unbiased estimator of the cross-correlation function.
+
+    Calculates the unbiased estimator of the cross-correlation function
+    R(tau) = 1/(N-|k|) R'(tau), where R'(tau) = E[x(t)*y(t+tau)] in a
```

Comment:
You could use `.. math::` for the mathematical formulations to have them in a nice format on the docs page. 
```suggestion
    .. math:: R(tau) = frac{1}/{(N-|k|)} R^{\prime}(tau), \text{where} R^{\prime}(tau) = E[x(t)*y(t+tau)] 
```
Please have a look at section 12 of the [numpy docs](https://numpydoc.readthedocs.io/en/latest/format.html) for possibilities.

Reply:
Good idea and I tried it. Unfortunately, the latest Spyder version 3.2.6 cannot render this and the output becomes shitty (see below). I would therefore suggest not to write it like this.

--- docstring output ---

Calculates the unbiased estimator of the cross-correlation function [1]

.. math:: R(        au) = rac{1}{N-|k|} R'(        au) \ ,

ight>` in a pairwise manner,eft<x(t)y(t+    au)
i.e. signal[ch_pairs[0,0]] vs signal2[ch_pairs[0,1]], signal[ch_pairs[1,0]]
vs signal2[ch_pairs[1,1]], and so on. The cross-correlation function is
obtained by scipy.signal.fftconvolve. Time series in signal are zscored
beforehand. Alternatively returns the Hilbert envelope of :math:`R( au)`,
which is useful to determine the correlation length of oscillatory signals.

[1] Hall & River (2009) "Spectral Analysis of Signals, Spectral Element
Method in Structural Dynamics", Eq. 2.2.3